11-multi-track/bb_models_collections/readme.md

```diff
@@ -347,26 +390,28 @@ var Grumble = Backbone.Model.extend({
 
```

Comment:
Can we use a relative url here like we do in the collection later on (i.e., no `http://localhost:3000`)?